worker/pyproject.toml

```diff
@@ -43,7 +43,7 @@ test = ["coverage", "pytest", "pytest-cov"]
 "Homepage" = "https://netboxlabs.com/"
 
 [project.scripts]
-worker = "worker.main:main"
+orb-worker = "worker.main:main"
```

Comment:
I think we have also README with example usage referring to `worker` to update

Reply:
yes! thanks for that

Reply:
Done!